pkg/cmd/cli/cmd/projects.go

```diff
@@ -99,8 +113,8 @@ func (o ProjectsOptions) RunProjects() error {
 	currentContext := config.Contexts[config.CurrentContext]
 	currentProject := currentContext.Namespace
 
-	var currentProjectExists bool = false
-	var currentProjectErr error = nil
+	var currentProjectExists bool
+	var currentProjectErr error
```

Comment:
Just for the future, prefer using: `currentProjectExists := nil` than this, because it immediately gives you the default value of a variable and it's less typing ;) Besides if you have two or more vars use block declaration:
```
var (
    currentProjectExists bool
    currentProjectErr error
)
```